### ISSUE TITLE
SALib.sample.sobol.sample returns ValueError: skip_values must be a positive integer

### ISSUE THREAD

**Issue by schwemro** [SALib/SALib#543]
The example below causes a `ValueError: skip_values must be a positive integer.`:
```
from SALib.sample import sobol
bounds = {'num_vars': 3, 'names': ['b_transp', 'a_q_rz', 'a_q_ss'], 'bounds': [[1, 50], [1, 50], [1, 50]]}
params = sobol.sample(bounds, 1024, calc_second_order=False)
```

System: Darwin
Python: 3.9.13
salib: 1.4.6

**Comment by tupui** [comment]
Hi @schwemro thank you for reporting. This was just fixed and we already published a new version on PyPi.

**Comment by ConnectedSystems** [comment]
Released v1.4.6.1 which includes the hotfix (your PR @tupui).

Next release should include tests to cover this case, and associated usage via the CLI

Thanks again @schwemro for reporting the issue.
